packages/cli/src/PublicApi/v1/handlers/executions/executions.service.ts

```diff
@@ -42,15 +34,43 @@ export async function getExecutions(params: {
 		where.id = Not(In(params.excludedExecutionsIds));
 	}
 
-	if (params.status) {
-		where = { ...where, ...getStatusCondition(params.status) };
-	}
 
 	if (params.workflowIds) {
 		where = { ...where, workflowId: In(params.workflowIds) };
 	}
+	if (!params.status) {
+		return Container.get(ExecutionRepository).findMultipleExecutions(
+			{
+				select: [
+					'id',
+					'mode',
+					'retryOf',
+					'retrySuccessId',
+					'startedAt',
+					'stoppedAt',
+					'workflowId',
+					'waitTill',
+					'finished',
+				],
+				where,
+				order: { id: 'DESC' },
+				take: params.limit,
+				relations: ['executionData'],
+			},
+			{
+				includeData: params.includeData,
+				unflattenData: true,
+			},
+		);	
+	}
+	if (typeof(params.status) == "string"){
+		where.status = params.status;
+	}
+	else {
+		where.status = In(params.status)
+	}
```

Comment:
I don't get why we need to add these lines. Can't we juste update the `getStatusCondition` ?

we previously had only these lines to handle if status is present or not. Can't we reuse the same logic ?

```
if (params.status) {
		where = { ...where, ...getStatusCondition(params.status) };
	}
```